base/src/main/java/uk/ac/ebi/atlas/search/suggester/SuggesterService.java

```diff
@@ -22,10 +28,24 @@ public SuggesterService(SuggesterDao suggesterDao, SpeciesFactory speciesFactory
         this.speciesFactory = speciesFactory;
     }
 
-    public Stream<Map<String, String>> fetchGroupedIdSuggestions(String query, String...  species) {
+    public Stream<Map<String, String>> fetchPropertiesWithoutHighlighting(String query, String...  species) {
         Species[] speciesArray = Arrays.stream(species).map(speciesFactory::create).toArray(Species[]::new);
 
         return suggesterDao.fetchBioentityProperties(query, DEFAULT_MAX_NUMBER_OF_SUGGESTIONS, false, speciesArray)
-                .map(suggestion -> ImmutableMap.of("term", suggestion.getTerm(), "category", suggestion.getPayload()));
+                .map(SUGGESTION_TO_MAP);
+    }
+
+    public Stream<Map<String, String>> fetchPropertiesWithHighlighting(String query, String...  species) {
```

Comment:
Could you maybe add a comment here saying what the difference between highlighting and no highlighting is? Because there is nothing in the return type that suggests a difference, and it is a very Solr-specific behaviour.